src/arguments/utility.py

```diff
@@ -103,13 +115,27 @@ def get_ans(self, questions_list):
                 sys.exit()
             json_ans_data = resp.json()
 
-            for data in json_ans_data["items"]:
-                output_content = [
-                    colored(
-                        "--------------------------------------------------------",
-                        'red'), data["body_markdown"],
-                    f"Link to the answer:{data['link']}"
-                ]
+            answers = json_ans_data["items"]
+            
+
+            #loop that handles each Answers
+
+            i = 0
+            run = True
+
+            while run:
+
+                self.clear()
```

Comment:
Another problem is that you don't allow user to read the answer, the answer just keeps shifting as soon as the answer get's displayed

Reply:
No, that's not true. The code allow user to navigate through the answers with the help of their keyboard. The Gif is fast forwarded, that's why it may look like it.